muddy_macros/src/internal.rs

```diff
@@ -38,14 +38,13 @@ pub fn build_obfuscation_mod(
                 let _ = write!(out, "{c:02X}");
                 out
             });
-            let env_name = env.as_ref().map_or("MUDDY", |s| s.as_str());
-            eprintln!("Please, set {env_name} env variable with content:");
+            let env_name = env.as_ref().map_or(DEFAULT_ENV, |s| s.as_str());
             #[cfg(windows)]
             // language=cmd
-            eprintln!(r#"set "{env_name}={key}""#);
+            eprintln!(r#"set {env_name}="{key}""#);
```

Comment:
It's a wrong example of usage for Windows:

```cmd
set {env_name}="{key}"
```

Because an environment variable value will be contain quotes (`"`) on each side, like:

```
"value"
```

Correct example:

```cmd
set "{env_name}={key}"
```

It's works for values with and without spaces.

Reply:
Thanks for the catch! Fixing this now